specification.md

```diff
@@ -524,9 +526,11 @@ enum AgentCapabilities {
     ReportsOwnLogs                 = 0x00000080;
```

Comment:
I see `ReportsOwn*` marked beta in proto but not here, is this intended?

Reply:
Fixed.